supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -219,7 +243,12 @@ object JesApiQueryManager {
     */
   final case class DoCreateRun(workflowId: WorkflowId, genomicsInterface: Genomics, rpr: RunPipelineRequest) extends JesApiQueryManagerRequest
 
-  private[statuspolling] trait JesApiQuery {
+  /**
+    * Abort a pipeline.
+    */
+  final case class DoAbortRun(workflowId: WorkflowId, run: Run) extends JesApiQueryManagerRequest
+
+  private[statuspolling] sealed trait JesApiQuery {
```

Comment:
TOLish and obviously preexisting but the use of "Query" terminology in this API for state-changing operations is confusing, particularly when the more accurate "request" is used internally